Sources/HaCWebsiteLib/Models/NewWorkshop.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+import Yaml
+
+/// Represents a workshop
+// TODO: Rename this struct to `Workshop` when it's ready for use
+public struct NewWorkshop {
+  let title: String
+
+  /// People that have directly contributed content to this workshop
+  let contributors: [String]
+
+  /// Non-contributors we would like to thank for helping to inspire, develop or facilitate this workshop
+  let thanks: [String]
+
+  /// Links that attendees may which to read after the workhsop
+  let furtherReadingLinks: [(String, URL)]
+
+  /// A link to a video recording of this workshop
+  let recordingLink: URL?
+
+  /// A link to any slides for this workshop
+  let slidesLink: URL?
+
+  /// A set of keywords associated with the workshop
+  let tags: [String]
```

Comment:
I was actually thinking about this a bit, we should probably make sure that when this system goes live, we have a `/tags` page that allows us to view all the tags created so far. That way when tagging a workshop as that list grows, people can look at the ones that exist so that they use the `canonical` versions.

Reply:
Sounds easy enough 👍

Reply:
See issue #165, just created it.